modules/table/src/test/java/org/apache/ignite/internal/table/distributed/TableManagerRecoveryTest.java

```diff
@@ -206,6 +207,7 @@ void after() throws Exception {
     }
 
     @Test
+    @WithSystemProperty(key = COLOCATION_FEATURE_FLAG, value = "false")
     public void testTableIgnoredOnRecovery() throws Exception {
```

Comment:
Why it's !colocation only? 
I guess it should have `@WithSystemProperty(key = COLOCATION_FEATURE_FLAG, value = "false")` until https://issues.apache.org/jira/browse/IGNITE-24397 is implemented.

Reply:
Same is above: this test makes checks for actions that we don't do for colocation case anymore

Reply:
Fixed, see https://github.com/apache/ignite-3/pull/5391#discussion_r1993686745